stable/ksoc-plugins/templates/eks_addon_deployment.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.eksAddon.enabled  }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ksoc-eks-marketplace-deployment
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app_name: ksoc-eks-marketplace-deployment
+spec:
+  replicas: 0
```

Comment:
`replicas: 0` is hardcoded. Is it ok?

Reply:
I think it is. This was the example that was given by our AWS rep.